src/routes/data-privacy.svx

```diff
@@ -0,0 +1,138 @@
+---
+layout: learn-more
+slug: data-privacy
+---
+
+<h5>Who we are</h5>
```

Comment:
Umm... it's raw HTML. Can we convert it into Markdown instead, since we're using Markdown preprocessor already? Otherwise, we could just use simple components.

Reply:
yup, will convert it